Makefile

```diff
@@ -114,6 +117,10 @@ docker-build:
 
 	@test -s ./overlays/dev/manager_image_patch.yaml || bash -c 'echo -e ${MANAGER_IMAGE_PATCH} > ./overlays/dev/manager_image_patch.yaml'
 
+ifeq ($(TRAVIS),true)
```

Comment:
I'm not super enthusiastic about adding a switch in the middle of a rule, but I can't think of anything better.

Reply:
likewise. let me know if anyone can think of a better solution, happy to change